.pre-commit-config.yaml

```
@@ -32,3 +32,10 @@ repos:
     rev: 1.13.0
     hooks:
       - id: blacken-docs
+  - repo: https://github.com/pycqa/pydocstyle
+    rev: 6.3.0
+    hooks:
+      - id: pydocstyle
+        args:
+          - --select=D103,D200,D206,D300,D301
+        files: src/*
```

```suggestion
        files: ^src/
```
Executing it locally I was getting a warning that is removed in this way

---

Thanks :)